### PR TITLE
Fix `mypy` now that the refactor PR has been merged

### PR DESCRIPTION
**Pull request by paddyroddy**
Starting https://github.com/brainglobe/cellfinder-napari/pull/65 afresh

### CODE REVIEW

File: cellfinder_napari/detect.py [hunk 1]
```diff
@@ -63,15 +63,15 @@ def widget(
...
+        trained_model: Optional[Path],
         misc_options,
         start_plane: int,
         end_plane: int,
         n_free_cpus: int,
         analyse_local: bool,
         debug: bool,
         reset_button,
-    ) -> List[napari.types.LayerDataTuple]:
+    ) -> None:
```
Comment (paddyroddy) [inline cellfinder_napari/detect.py:74]: This appears to have been here prior to #54, but the function doesn't actually return anything

File: cellfinder_napari/train.py [hunk 2]
```diff
@@ -183,7 +184,7 @@ def widget(
         if Model_weights == Path.home():
             Model_weights = None
 
-        if YAML_files[0] == Path.home():
+        if YAML_files[0] == Path.home():  # type: ignore
```
Comment (paddyroddy) [inline cellfinder_napari/train.py:187]: I can fix this in #56, but for now this will pass `mypy`

Comment (alessandrofelder) [reply cellfinder_napari/train.py:187]: Cool!

File: cellfinder_napari/detect.py (hunk 1 above)
Comment (alessandrofelder) [reply cellfinder_napari/detect.py:74]: That's right - good catch.

From looking at the git blame, the return type changed [in a refactoring commit](https://github.com/brainglobe/cellfinder-napari/commit/1ba0ac6abd9028e57e0d659e9d243849ebc96851) but the type hint wasn't adapted. Before, the two points layers were returned - After, they were added directly to the viewer in the `add_layers` callback function.

**Review by alessandrofelder — APPROVED** [review]
Not sure if this is ready to review yet, as still marked as draft, but at least so far, looks great to me! 
Thanks @paddyroddy - happy to see the CI pass :) if you make more changes, feel free to ask for my review again.

**Comment by paddyroddy** [comment]
> Not sure if this is ready to review yet, as still marked as draft, but at least so far, looks great to me! Thanks @paddyroddy - happy to see the CI pass :) if you make more changes, feel free to ask for my review again.

yep sorry this was done, first time I've used the draft feature

**Pull request merged**